lib/mapper.js

```diff
@@ -255,25 +274,16 @@ function formatMetadata(props){
       ret[dateKey(key)] = time(val);
       return;
     }
-
+    if (is.array(val)) {
+      // Will flatten later
+      ret[key] = val;
+    }
     if (is.object(val)) {
-      if (!~safe.indexOf(Object.keys(val))) {
```

Comment:
The `safe` array that this was referencing is no longer used, so we can remove it.